base/redis/redis-cache.Deployment.yaml

```diff
@@ -36,15 +36,17 @@ spec:
         - containerPort: 6379
           name: redis
         readinessProbe:
-          initialDelaySeconds: 5
+          initialDelaySeconds: 10
+          timeoutSeconds: 5
```

Comment:
For consistency with cloud: https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/release/base/redis/redis-store.Deployment.yaml#L61